src/stylesheets/components/_header.scss

```diff
@@ -193,7 +193,7 @@ $z-index-nav:     9000;
       &[aria-expanded=false],
       &[aria-expanded=true] { /* stylelint-disable-line selector-no-qualifying-type */
         @include media($nav-width) { /* stylelint-disable-line selector-no-qualifying-type */
-          background-position: right 1.5rem top 50%;
+          background-position: right 1.5rem top 44%;
```

Comment:
![screen shot 2017-07-17 at 5 38 59 pm](https://user-images.githubusercontent.com/776987/28292881-dcbd1554-6b16-11e7-8438-4fe255d04d6b.png)

fwiw we can also adjust the bottom border on the span so it doesn't line up with the bottom of the nav if we want to fit the style but keep things centered.